voluncheer/.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 100
+exclude = */migrations/*
```

Comment:
*_initial.py should also be included? Those migration files are generated by the system.

Reply:
The only `*_initial.py` files I saw were within a `migrations/` directory so they are covered by this exclude. Do these files appear elsewhere?

Reply:
Good point.